cmd/jira/quick-task.go

```diff
@@ -27,10 +27,10 @@ var quickTaskCmd = &cobra.Command{
 The ticket will be assigned to the caller and added to their team's current sprint as an OSD Task.
 A link to the created ticket will be printed to the console.`,
 	Example: `#Create a new Jira issue
-osdctl jira create "Update command to take new flag"
+osdctl jira quick-task "Update command to take new flag"
```

Comment:
Found these while creating an accompanying card for this effort and figured I'd fix the typos here real quick.